pages/_app.js

```diff
@@ -1,59 +1,50 @@
 import GlobalStyle from "../styles";
 import Layout from "@/components/layout";
-import { ideas as defaultIdeas } from "@/lib/db";
-import { useState } from "react";
-import { v4 as uuidv4 } from "uuid";
-import { useRouter } from "next/router";
+import useSWR from "swr";
+import { SWRConfig } from "swr";
+
 import useLocalStorageState from "use-local-storage-state";
 
+const fetcher = (url) => fetch(url).then((response) => response.json());
+
 export default function App({ Component, pageProps }) {
-  const [ideas, setIdeas] = useLocalStorageState("ideas", {
-    defaultValue: defaultIdeas,
+  const [favourites, setFavourites] = useLocalStorageState("favourites", {
+    defaultValue: "",
```

Comment:
I assume this is supposed to be an array, so the `defaultValue` can also be an empty array to avoid errors elsewhere.

```suggestion
    defaultValue: [],
```